tests/namespace/test_lldp.py

```diff
@@ -124,11 +124,36 @@ def test_subtype_lldp_loc_man_addr_table(self):
 
 
     def test_subtype_lldp_rem_man_addr_table(self):
+        # Verfiy both valid ipv4 and ipv6 address exit
         for entry in range(3, 6):
             mib_entry = self.lut[(1, 0, 8802, 1, 1, 2, 1, 4, 2, 1, entry)]
             ret = mib_entry(sub_id=(1, 1))
             self.assertIsNotNone(ret)
             print(ret)
+        # Verfiy both valid ipv4 and invalid ipv6 address exit
+        for entry in range(3, 6):
+            mib_entry = self.lut[(1, 0, 8802, 1, 1, 2, 1, 4, 2, 1, entry)]
+            ret = mib_entry(sub_id=(1, 5))
+            self.assertIsNotNone(ret)
```

Comment:
How is the return value check if both ipv4 and ipv6 exits?

Reply:
intention of testcase is to check when both ip address are present we can get the data from first key (can be either v4 or v6) for given interface. Basically to check we are not ignoring management ip information in such case.

Reply:
@SuvarnaMeenakshi Updated test case for this. Fixed `lookup()` API.